### ISSUE TITLE
Add contribution guide

### ISSUE THREAD

**Issue by n1ru4l** [react-native-community/react-native-camera#1333]
I want to contribute. However there is no documentation around the tooling, especially commitlint? I cannot do any commit without bypassing the githooks. In what format should I do those?

Are there any other things I have to consider?

**Comment by sibelius** [comment]
we would accept a pr adding a CONTRIBUTION.md for sure

tldr about commitlint

#### Update CHANGELOG.md before Release

Everytime a new release needs builded, we must update the changelog. The changelog summary will be generated from a commit pattern.

```
type(category): description
```

Where `type` is one of the following:

* `build`
* `ci`
* `chore`
* `docs`
* `feat`
* `fix`
* `other`
* `perf`
* `refactor`
* `revert`
* `style`
* `test`

And `category` can be anything of your choice. If you use a type not found in the list (but it still follows the same format of the message), it'll be grouped under `other`.


**Example**

```
feat(landing-booking): new button to submit
```


##### Update only build

```sh
yarn release:build
```

##### Update build & patch

```sh
yarn release:patch
```

##### Update build & minor

```sh
yarn release:minor
```

##### Update build & major

```sh
yarn release:major
```

**Comment by n1ru4l** [comment]
Thank you ☺️
